ci/Makefile

```diff
@@ -48,9 +48,13 @@ test_jenkins_jobs:
 update_jenkins_jobs:
 	jenkins-jobs  --ignore-cache --conf ${JENKINS_JOB_CONFIG} update ${JENKINS_JOBS_DIR}/:${JENKINS_JOBS_DIR}/templates/
 
+.PHONY: update_pkgs
+update_pkgs:
+	sudo zypper update --allow-vendor-change -y terraform terraform-provider-*
```

Comment:
Why only terraform? I think we should include kubernetes-client too

Reply:
I was focusing on terraform, but makes sense. What about docker?